example/imagery/plugin.js

```diff
@@ -129,6 +129,25 @@ define([
                                 hints: {
                                     image: 1
                                 }
+                            },
+                            {
+                                key: "layers",
```

Comment:
FYI: This metadata needs discussion - @akhenry has suggested creating a new type of metadata called "layers".
Using enumerations for now.